test/Makefile.mk

```
@@ -192,6 +192,10 @@ GTEST_TESTS = test/gtest_links/AccumulatorTest.empty \
               test/gtest_links/FrequencyMapAgentTest.policy_to_json \
               test/gtest_links/FrequencyMapAgentTest.split_policy \
               test/gtest_links/FrequencyMapAgentTest.validate_policy \
+              test/gtest_links/FrequencyMapAgentTest.test_hash_freq_map \
+              test/gtest_links/FrequencyMapAgentTest.test_default_freq_hash \
+              test/gtest_links/FrequencyMapAgentTest.test_both_map_and_set \
+              test/gtest_links/FrequencyMapAgentTest.test_neither_map_nor_set \
```

```suggestion
              test/gtest_links/FrequencyMapAgentTest.report_hash_freq_map \
              test/gtest_links/FrequencyMapAgentTest.report_default_freq_hash \
              test/gtest_links/FrequencyMapAgentTest.report_both_map_and_set \
              test/gtest_links/FrequencyMapAgentTest.report_neither_map_nor_set \
```

Then alphabetize the list above here.